understanding/22/target-size-minimum.html

```diff
@@ -116,8 +117,8 @@ <h2>Benefits of Target Size (Minimum)</h2>
         <h2>Examples of Target Size (Minimum)</h2>
 
         <ul>
-            <li>Three buttons are on-screen and the target area of each button is 24 by 24 CSS pixels. Since the target size itself is 24 x 24 px, no additional spacing is required, the Success Criterion passes.</li>
-            <li>A row of buttons, each of which has a horizontal width of more than 50px, but a height of only 20 px. Since there is sufficient spacing above and below the row of buttons, the Success Criterion passes.</li>
+            <li>Three buttons are on-screen and the target area of each button is 24 by 24 CSS pixels. Since the target size itself is 24 x 24 CSS pixels, no additional spacing is required, the Success Criterion passes.</li>
```

Comment:
Still has an instance of "24 x 24 CSS pixels" rather than "24 by 24 CSS pixels"

Reply:
fixed, thanks.

Reply:
i still see it in here "Since the target size itself is 24 x 24 CSS pixels, no additional spacing is required"